portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,95 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Comment {
+    private final String text;
+    private final String user;
+    private final long timestamp;
+    private int upvotes;
+    private int downvotes;
+    private long key;
+
+    //Constructor to use when comment is created
```

Comment:
[nit] Leave a space between the // and the start of the comment: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments